### PR TITLE
feat: add Actor Standby templates

### PR DESCRIPTION
**Pull request by jirimoravcik**
This PR adds templates for Actor Standby.

### CODE REVIEW

**Comment by jirimoravcik** [comment]
Converted to a draft as we need to add `usesStandbyMode` logic that would setup Standby for the Actors

File: templates/js-standby/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Empty JavaScript template
+
+Start a new [web scraping](https://apify.com/web-scraping) project quickly and easily in JavaScript (Node.js) with our Standby project template. It provides a basic structure for building an Actor with [Apify SDK](https://docs.apify.com/sdk/js/) and allows you to easily add your own functionality.
+
+## Included features
```
Comment (fnesveda) [inline templates/js-standby/README.md:5]: Let's link Actor Standby docs here too, same in the other templates

File: templates/js-standby/.actor/actor.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "actorSpecification": 1,
+    "name": "js-standby",
+    "title": "Standby Actor in JavaScript",
+    "description": "Standby Actor in JavaScript.",
+    "version": "0.0",
+    "buildTag": "latest",
+    "meta": {
```
Comment (fnesveda) [inline templates/js-standby/.actor/actor.json:8]: Let's add the `usesStandbyMode` top level property as per discussion in Slack. Also to all the other templates.
```suggestion
    "usesStandbyMode": true,
    "meta": {
```

**Review by fnesveda — CHANGES_REQUESTED** [review]
Looks good, two small notes

**Comment by drobnikj** [comment]
Can I go for a review? It is in draft

**Comment by jirimoravcik** [comment]
> @jirimoravcik Can I go for a review? It is in draft

go for it, corresponding API change in https://github.com/apify/apify-core/pull/16649

File: templates/js-standby/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "js-standby-project",
+    "version": "0.0.1",
+    "type": "module",
+    "description": "This is a boilerplate of an Apify Actor.",
+    "engines": {
+        "node": ">=18.0.0"
```
Comment (drobnikj) [inline templates/js-standby/package.json:7]: Is it necessary to have a node js version here? If so, I would put at least the v20 as a docker image.

File: templates/ts-standby/package.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "ts-standby-project",
+    "version": "0.0.1",
+    "type": "module",
+    "description": "This is an example of an Apify actor.",
+    "engines": {
+        "node": ">=18.0.0"
```
Comment (drobnikj) [inline templates/ts-standby/package.json:7]: maybe same here

File: templates/ts-standby/package.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "ts-standby-project",
+    "version": "0.0.1",
+    "type": "module",
+    "description": "This is an example of an Apify actor.",
```
Comment (drobnikj) [inline templates/ts-standby/package.json:5]: nit: little bit tune description about standby

File: templates/js-standby/package.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "js-standby-project",
+    "version": "0.0.1",
+    "type": "module",
+    "description": "This is a boilerplate of an Apify Actor.",
```
Comment (drobnikj) [inline templates/js-standby/package.json:5]: nit: little bit tune description about standby

**Review by drobnikj — APPROVED** [review]
Looks good just a few minor stuff

**Comment by jirimoravcik** [comment]
I'll finalize this after https://github.com/apify/apify-docs/pull/1096 is merged so it can be linked

**Review by fnesveda — APPROVED** [review]

**Pull request merged**